### DISCUSSION TITLE
Can't find pull request preview

### DISCUSSION THREAD

**Discussion by jquentin-lion** [GitbookIO/community#805] [0. General bugs and questions]
I don't understand this [paragraph of the documentation](https://docs.gitbook.com/integrations/git-sync/github-pull-request-preview#security-considerations) about Pull Requests previews.

> For every PR done using a target branch synced with a GitBook space, you’ll see a status added to the PR with a unique preview URL. Clicking the Details link on the status will take you to the preview URL for your content.

Where am I supposed to see this status? In the gitbook interface? Github?

I have created a pull request and I don't see any preview button anywhere.

(The synced branch of my gitbook space is main, the PR targets merging into main, my github sync config confirms it does have pull requests read access in that repo)

**Comment by aodonnell-ca** [comment]
I am seeing the same issue. The documentation makes it sound (to me) like the preview should be accessible from within GitHub.

I did find https://docs.gitbook.com/getting-started/git-sync/troubleshooting#github-preview-is-not-showing, but since I'm new to GitBooks, then my integration was newer than the listed date. I uninstalled and re-installed it to make sure.

My repo is private and unpaid. My GitBook is public and also using the "free" version.

Do the instructions for preview only apply at a certain subscription level? I would be okay with that, but the instructions don't say that is a requirement.
